PRFCS/prfc-2.md

```diff
@@ -26,6 +26,12 @@ The 'Required Calls' section lists the set of methods that all smart contracts t
 
 **Exclusive Spender**: An exclusive spender is an account that is approved to transfer *all* tokens owned by the owner. An account can be made an Spender of all tokens through a single call to `set_exclusive_spender`.
 
+## Notes
+
+- The following uses syntax from Rust (version 1.59.0).
+- The data type `PublicAddress` in this context is the type alias to 32-byte slice `[u8; 32]`. 
```

Comment:
Do we really need "in this context"? There is no other context where 'PublicAddress' is something else right?